File: Project Euler Archives/1. Multiples of 3 or 5/C++/solution.cpp
#include <iostream>

using namespace std;

int main()
{
    int ans=0;
    for (int i = 0; i < 1000; i++){
        if (i % 3 == 0 || i % 5 == 0){
            ans += i;
        }
    }
    cout<<ans;

    return 0;
}

//time complexity = O(n)
//space complexity = O(n)

File: Project Euler Archives/1. Multiples of 3 or 5/Java/solution.java
public class Main
{
	public static void main(String[] args) {
    int i, ans=0;
    for (i = 0; i < 1000; i++){
        if (i % 3 == 0 || i % 5 == 0){
            ans += i;
        }
    }
    System.out.println(ans);
    }
}
